Minio/MinioClient.cs

```diff
@@ -70,7 +72,10 @@ public partial class MinioClient
         // Enables HTTP tracing if set to true
         private bool trace = false;
 
+        private int requestTimeout;
+
         private const string RegistryAuthHeaderKey = "X-Registry-Auth";
+        private HttpClient HttpClient { get; } = new HttpClient();
```

Comment:
can I suggest we make this injectable through constructor? for testing/logging purposes

Reply:
Thanks @chester89,
Your suggestion is already part of this PR, #551.